.pre-commit-config.yaml

```diff
@@ -51,6 +51,8 @@ repos:
     rev: 0.7.16
     hooks:
       - id: mdformat
+        additional_dependencies:
+          [mdformat-gfm, mdformat-frontmatter, mdformat-footnote]
```

Comment:
Wondering if it's possible to add these as `dev` dependencies.

With `prettier` it was possible to format markdown files automatically in IDEs like VS Code which was quite handy. I don't know if it's also done automatically with `mdformat` or if it needs additional IDE configuration?

Reply:
Perhaps we should stick with Prettier for all but the changelog?

Reply:
Yeah maybe. Anyway we can have a look separately.